README.md

```diff
@@ -1,94 +1,92 @@
-# Ibus-typing-booster
+ibus-typing-booster
```

Comment:
https://guides.github.com/features/mastering-markdown/
```
Headers

# This is an <h1> tag
## This is an <h2> tag
###### This is an <h6> tag
```
So why are you removing the #?

Reply:
Oh, oops! my mistake.